modules/ROOT/pages/_partials/endpoint-as2-send.adoc

```diff
@@ -0,0 +1,227 @@
+== AS2 Send Settings
+
+AS2 Send Settings pertain to trading partner-specific AS2 settings such as *certificate* and *URL*.
+
+[%header,cols="3s,7a"]
+|===
+|Setting |Description
+
+| URL
+
+|The HTTP/S address at which your partner is accepting incoming AS2 messages. +
+
+
+Required?::
+Yes, if AS2 is the transport protocol
+
+Valid values::
+
+Alphanumeric
+
+Default::
+
+None
+
+Example::
+`www.midsizesupplier.com:8080`
+
+| Subject
+
+|The field that specifies a default subject for messages you send out to this partner. +
+
+Required?::
+Yes
+
+Valid values::
+
+Alphanumeric
+
+Default::
+`None`
+
+| Encoding
+
+| Type of encoding to use when sending a message with AS2. +
+
+Required::
+Yes
+
+Default::
+Binary
+
+Valid values::
+
+* *Binary* +
+A method of encoding in which a message is represented using a combination of 0s and 1s.
+
+* *7-bit*
++
+Data that is all represented as relatively short lines with 998 bytes or less between CRLF line separation sequences.
+No bytes with decimal values greater than 127 are allowed and neither are NULLs (bytes with decimal value 0).
+CR (decimal value 13) and LF (decimal value 10) bytes only occur as part of CRLF line separation sequences.
+
+* *8-bit*
++
+Data that is all represented as relatively
+short lines with 998 bytes or less between CRLF line separation
+sequences. Bytes with decimal values greater than 127
+can be used.  As with "7bit data" CR and LF bytes only occur as part
+of CRLF line separation sequences. No NULLs are allowed.
+
+* *Base 64*
++
+Need info here.
+
+* *Quoted-Printable*
++
+Intended to represent data that largely consists of bytes that correspond to printable characters in
+the US-ASCII character set.  It encodes the data in such a way that
+the resulting bytes are unlikely to be modified by mail transport.
+If the data being encoded are mostly US-ASCII text, the encoded form
+of the data remains largely readable.  A body that is
+entirely in the US-ASCII character set may also be encoded in Quoted-Printable to ensure
+the integrity of the data should the message pass through a
+character-translating, and/or line-wrapping gateway.
+
+| Message Encryption
+
+| The algorithm used to encrypt “send” messages. +
```

Comment:
Why is send in quotes?

Reply:
Probably a mistake on my part, will double check.